From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Johnson <johnson.peter@gmail.com>
Date: Sun, 27 Jul 2014 23:59:59 -0700
Subject: [PATCH 0207/6262] ADXL345_I2C: Make constructor explicit.

The second argument has a default value, so it needs to be explicit.

Change-Id: I5c38f8f55f9c21576835c005acd8d80fe6864140
---
 wpilibc/wpilibC++/include/ADXL345_I2C.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpilibc/wpilibC++/include/ADXL345_I2C.h b/wpilibc/wpilibC++/include/ADXL345_I2C.h
index 86fe41c40743ee1633cceec44b3f0f37f77769dd..5cff3355f03e5e852076f48f834994fb450d12ec 100644
--- a/wpilibc/wpilibC++/include/ADXL345_I2C.h
+++ b/wpilibc/wpilibC++/include/ADXL345_I2C.h
@@ -36,7 +36,7 @@ public:
 	};
 
 public:
-	ADXL345_I2C(Port port, Range range = kRange_2G);
+	explicit ADXL345_I2C(Port port, Range range = kRange_2G);
 	virtual ~ADXL345_I2C();
 
 	// Accelerometer interface
